provider/cmd/pulumi-resource-docker/schema.json

```diff
@@ -3629,7 +3629,7 @@
+            "description": "\u003c!-- Bug: Type and Name are switched --\u003e\nManages the lifecycle of docker image in a registry. You can upload images to a registry (= `docker push`) and also delete them again\n\n",
```

Comment:
@t0yv0 is it expected for some of these examples to be dropped? This seems like a step backward, but maybe they were just malformed or didn't convert cleanly?